Bricks/file/file.h

```diff
@@ -214,79 +214,125 @@ struct FileSystem {
   // TODO(dkorolev): Make OutputFile not as tightly coupled with std::ofstream as it is now.
   typedef std::ofstream OutputFile;
 
+  struct ScanDirItemInfo {
+    const std::string name;
+    const std::string path;
+    const bool is_directory;
```

Comment:
Usually, the members of such structures don't have `const` qualifiers and the const-ness is controlled for the particular instance of a structure.